catalogs/sources/gtfs/schedule/us-kansas-salina-gtfs-1867.json

```diff
@@ -0,0 +1,21 @@
+{
+    "mdb_source_id": 1867,
+    "data_type": "gtfs",
+    "provider": "81 Connection, CityGo, KanConnect",
+    "location": {
+        "country_code": "US",
+        "subdivision_name": "Kansas",
+        "municipality": "Salina",
+        "bounding_box": {
+            "minimum_latitude": null,
+            "maximum_latitude": null,
+            "minimum_longitude": null,
+            "maximum_longitude": null,
```

Comment:
null bbox?

Reply:
Yes, this is due to [the SSL certificate error ](https://github.com/MobilityData/mobility-database-catalogs/issues/216) we currently get on specific feeds — this feed had to be added manually, which means the bounding box isn't generated.